server/autonomyclass.py

```
@@ -0,0 +1,118 @@
+        )
+
+        self._executor = SingleThreadedExecutor()
+        self._executor.add_node(self._node)
+
+        self._thread = threading.Thread(target=self._spin, daemon=True)
+
+    def _spin(self):
+        try:
+            self._executor.spin()
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)